### PR TITLE
Change description for ROLIST2 and other fixes

### PR DESCRIPTION
**Pull request by slavaleleka**
- change description for ROLIST2
- mark `Filtros Nauscopicos` and `Korean Adblock List` as obsolete
- update subscriptionUrl  for `Xfiles`
- change description for `Polskie Filtry Elementów Irytujących` — there are non-English descriptions for other [Polish filters](https://uploads.adguard.com/slbdzd8lzc7h5.png) or [`RU AdList`, for example](https://uploads.adguard.com/slbdzd8l1ozjl.png) 
- mention that some filters are already included into other ones (EasyList, AdGuard Safari filter, EasyList Dutch, EasyList Germany, Liste FR)
- typos

### CODE REVIEW

File: filters/filter_4_Social/metadata.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "filterId": 4,
   "name": "AdGuard Social Media filter",
-  "description": "Filter for social media widgets (Like buttons and such)",
+  "description": "Filter for social media widgets such as “Like” and “Share” buttons and more.",
```
Comment (Alex-302) [inline filters/filter_4_Social/metadata.json]: better replace quotation mark to '...' or "…"

File: filters/ThirdParty/filter_202_FiltrosNauscopicos/metadata.json [hunk 2]
```diff
@@ -1,15 +1,17 @@
 {
   "filterId": 202,
-  "name": "Filtros Nauscopicos",
+  "name": "(Obsolete) Filtros Nauscopicos",
   "description": "español",
```
Comment (Alex-302) [inline filters/ThirdParty/filter_202_FiltrosNauscopicos/metadata.json]: add description

File: filters/ThirdParty/filter_206_Xfiles/metadata.json [hunk 3]
```diff
@@ -1,13 +1,13 @@
 {
   "filterId": 206,
   "name": "Xfiles",
-  "description": "italiano",
+  "description": "Italiano adblock filter list",
```
Comment (Alex-302) [inline filters/ThirdParty/filter_206_Xfiles/metadata.json]: a dot at the end

File: filters/ThirdParty/filter_203_hufilter/metadata.json [hunk 4]
```diff
@@ -1,7 +1,7 @@
 {
   "filterId": 203,
   "name": "hufilter",
-  "description": "magyar",
+  "description": "Magyar adblock filter list",
```
Comment (AdamWr) [inline filters/ThirdParty/filter_203_hufilter/metadata.json]: I guess, that there should be a dot at the end too.

**Pull request merged**